cmake/FindCAN_Stack.cmake

```
@@ -3,6 +3,6 @@ if(NOT TARGET isobus::isobus)
   FetchContent_Declare(
     CAN_Stack
     GIT_REPOSITORY https://github.com/Open-Agriculture/AgIsoStack-plus-plus.git
-    GIT_TAG 30e51211a760c2df064dc2934a4f180c4cad73cc)
+    GIT_TAG 8ed87459e92448b8256a213f71379b266bf2af88)
```

```suggestion
    GIT_TAG 13470f33b2aa7fad274e8b3f96289566a0093b53)
```
If you update the stack then this is good to go.